### ISSUE TITLE
1.17 seems to break everything on iOS 11

### ISSUE THREAD

**Issue by jkichline** [parse-community/Parse-SDK-iOS-OSX#1277]
I had just updated my CocoaPods (iOS) and it installed Parse 1.17.0 (from 1.16.0) and nothing worked.  All queries would return nil as the result (array or object) and nil as an error.  I used Charles to monitor traffic and it appeared that the query was never sent!  It may have something to do with the user not being signed in prior to the query (since I'm trying to sign in but need to run a non-ACL query.  I reverted back to 1.16 and everything worked as before.

Can you please check to see if this is a known issue?  I get no error messages at all... just no data and no error.

**Comment by flovilmart** [comment]
Can you post some sample code please? Are you using swift or objective C? Also the traces of Charles with obfuscated keys and appId would help.

**Comment by Skyb0rg** [comment]
Same here .... hole Project doesn´t work. Dont get any data.

**Comment by flovilmart** [comment]
Are you guys able to add the request interceptors as described in the wiki and post the contents?

**Comment by jaredalexander** [comment]
same. Swift.

**Comment by flovilmart** [comment]
Again, please provide the contents of the queries when using the interceptor

**Comment by jeffreyjackson** [comment]
same.  objective-c.  reverting to 1.16

**Comment by flovilmart** [comment]
Closing the issue as it’s lackin the require info to process properly. Please post request /responses from the interceptor as descibed in the wiki and we’ll reopen.

**Comment by flovilmart** [comment]
If is possible for any of you to confirme the `release-1.17.1`  branch fixes the issue?

in your Podfile:

```
pod 'Parse', :git => 'https://github.com/parse-community/Parse-SDK-iOS-OSX', :branch => 'release-1.17.1'
```

**Comment by flovilmart** [comment]
Can you provide the logs with interceptors setup as well as an example of the query you’re making? This is really crucial. I don’t see the issue anymore with the branch on queries

**Comment by flovilmart** [comment]
@nickseidel

**Comment by jkichline** [comment]
Florent:

The 1.17.1 build does seem to work correctly.

I would have sent Charles proxy information, but with 1.17, no attempts were being made to even reach the Parse server.  It was as if the Parse framework was detecting that it didn’t have the rights or reasons to make a call to the network although other calls were made like for app metrics and installations.

Jason Kichline

**Comment by flovilmart** [comment]
Let’s give us a few days before merging it in.

**Comment by felix-dumit** [comment]
I can also confirm 1.17.1 fixed this issue for me.

I was experiencing it if I called a query using any of the async methods (bolts or completion handler), but not using the sync `findObjects` method.

**Comment by flovilmart** [comment]
That’s good to know. Sorry for that bad release, I wasn’t expecting it to break at that time.

**Comment by natanrolnik** [comment]
Closing this issue as 1.17.1 was released yesterday.

**Comment by Ramesh23iphone** [comment]
-> No results matched the query. (Code: 101, Version: 1.17.0)

        let query = PFQuery(className:"userDataBase")
        query.getObjectInBackground(withId: objectId){(objects: PFObject?, error: Error? ) in


            if error != nil {
                print(error as Any)
            } else if let rating = objects {
                rating["Rating"] = Double(self.ratingSlider.value)

                rating.saveInBackground()
            }
        }

**Comment by TomWFox** [comment]
@Ramesh23iphone I'm not sure this error has any relevance to this issue, but the bug was fixed in 1.17.1 so you can just update to that.
